opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -252,6 +271,46 @@ def __repr__(self):
         )
 
 
+class SumObserver(Observer, metrics_api.SumObserver):
+    """See `opentelemetry.metrics.SumObserver`."""
+
+    def _validate_observe(
+        self, value: metrics_api.ValueT, key: Tuple[Tuple[str, str]] = None,
+    ):
+        if super()._validate_observe(value, key):
```

Comment:
nitpick, but I think this would be easier to read if you used this as a guard condition:

```
if super()....:
    return False
if (key is not None....:
    return False
return True
```

Seems like calling the superclass validator, then calling your guard conditions, fits naturally.